artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/Bindings.java

```diff
@@ -63,7 +63,5 @@ public interface Bindings extends UnproposalListener {
 
    int size();
 
-   default boolean contains(Class clazz) {
-      return false;
-   }
+   boolean hasLocalBinding();
```

Comment:
I know this will have compilation failures :(... I'm fixing and I will push another commit